samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget-e2etests/Steps/Pages/HomePageSteps.cs

```diff
@@ -50,10 +50,10 @@ public void ThenTheId_TokenIsShownAndNotEmpty()
             _homePageModel.ClaimIdTokenLabel.Text.Should().NotBeEmpty();
         }
 
-        [Then(@"the refresh_token is shown and not empty")]
-        public void ThenTheRefresh_TokenIsShownAndNotEmpty()
-        {
-            _homePageModel.ClaimRefreshTokenLabel.Text.Should().NotBeEmpty();
-        }
+        //[Then(@"the refresh_token is shown and not empty")]
+        //public void ThenTheRefresh_TokenIsShownAndNotEmpty()
+        //{
+        //    _homePageModel.ClaimRefreshTokenLabel.Text.Should().NotBeEmpty();
+        //}
```

Comment:
can this be removed?

Reply:
I decided to keep it, because we need to add it back when fixing "add support for arrays in env vars"